packages/bbui/package.json

```diff
@@ -98,8 +98,7 @@
           {
             "projects": [
               "@budibase/string-templates",
-              "@budibase/shared-core",
-              "@budibase/types"
```

Comment:
This is not a bbui dependency

Reply:
shared-core is imported in `src/helpers.js` so it is a dependency. It's only proxying a module from shared-core though, so it could be refactored to be removed, but right now it is a dependency.